app/reducers/views/channel.js

```diff
@@ -55,22 +55,24 @@ function handleSelectChannel(state, action) {
     return data;
 }
 
-function handleSetTempUploadFileForPostDraft(state, action) {
-    if (action.rootId) {
-        return state;
-    }
+export const handleSetTempUploadFilesForPostDraft = (() => {
+    return (state, action) => {
+        if (action.rootId) {
+            return state;
+        }
 
-    const tempFiles = action.clientIds.map((temp) => ({...temp, loading: true}));
-    const files = [
-        ...state[action.channelId]?.files || [],
-        ...tempFiles,
-    ];
+        const tempFiles = action.clientIds.map((temp) => ({...temp, loading: true}));
+        const files = [
+            ...state[action.channelId]?.files || [],
+            ...tempFiles,
+        ];
 
-    return {
-        ...state,
-        [action.channelId]: Object.assign({}, state[action.channelId], {files}),
+        return {
+            ...state,
+            [action.channelId]: Object.assign({}, state[action.channelId], {files}),
+        };
     };
-}
+})();
```

Comment:
I wonder if this change to IIFE is necessary. Same with the other change.

Reply:
I wanted to keep it in local scope and avoid unnecessary global function (the original function was private and had to expose somehow to test)

Reply:
why is this receiving a callback?? this looks totally wrong to me, you should just export the existing function

Reply:
the function was not exported as there wasn't a need for it,  nothing more.

Reply:
Updated